sympy/matrices/expressions/matexpr.py

```diff
@@ -135,18 +135,18 @@ def __rpow__(self, other):
         raise NotImplementedError("Matrix Power not defined")
 
     @_sympifyit('other', NotImplemented)
-    @call_highest_priority('__rdiv__')
-    def __div__(self, other):
+    @call_highest_priority('__rtruediv__')
+    def __truediv__(self, other):
         return self * other**S.NegativeOne
 
     @_sympifyit('other', NotImplemented)
-    @call_highest_priority('__div__')
-    def __rdiv__(self, other):
+    @call_highest_priority('__truediv__')
+    def __rtruediv__(self, other):
         raise NotImplementedError()
         #return MatMul(other, Pow(self, S.NegativeOne))
 
-    __truediv__ = __div__  # type: Callable[[MatrixExpr, Any], Any]
-    __rtruediv__ = __rdiv__  # type: Callable[[MatrixExpr, Any], Any]
+    __truediv__ = __truediv__  # type: Callable[[MatrixExpr, Any], Any]
+    __rtruediv__ = __rtruediv__  # type: Callable[[MatrixExpr, Any], Any]
```

Comment:
I guess this can just be deleted

Reply:
My regex missed this due to the trailing comments, good catch.